bool        request_control
bool        request_resume
bool        request_standby
bool        request_shutdown
bool        execute_waypoints
bool        set_waypoints
